### ISSUE TITLE
Net block crash when arrays are empty

### ISSUE THREAD

**Issue by cameronswinoga** [greshake/i3status-rust#922]
Sometimes :tm: when not connected to any networks the net block will crash with the following error:
```
Error in block 'net': Malformed JSON: [{
        "addr_info": [{}]
    },{
        "addr_info": [{}]
    },{
        "addr_info": [{}]
    }
]
```

Relevant config entries:
```
[[block]]
block = "net"
device = "wlp111s0"
format = "{ssid} {signal_strength} {ip}"
interval = 5

[[block]]
block = "net"
device = "eno1"
format = "{ip}"
interval = 5
```

**Comment by ammgws** [comment]
https://github.com/greshake/i3status-rust/pull/901 ?

**Comment by cameronswinoga** [comment]
Sorry, forgot to add that this is on 0b8a5cab37a (which has the fix from #901 )

**Comment by ammgws** [comment]
Hopefully this should be fixed by #926
